visual_behavior/data_access/loading.py

```diff
@@ -190,8 +217,8 @@ def get_filtered_ophys_experiment_table(include_failed_data=False, release_data_
     if release_data_only:
         # get cache from lims for data released on March 25th
         print('getting experiment table for March 25th release from lims')
-        cache = bpc.from_lims(data_release_date='2021-03-25')
-        # cache = get_visual_behavior_cache(from_s3=False, release_data_only=True)
+        cache = bpc.from_lims(data_release_date=['2021-03-25', '2021-08-12'])
+        print('use get_released_ophys_experiment_table to get August release data in addition to March release')
```

Comment:
The comment seems to be out-of-date with the line above (220)

Reply:
thanks, will change